modules/ComscoreStreamingTag/tests/test_ads_2.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+	<title>ComscoreStreamingTag Module - ads </title>
+	<script type="text/javascript" src="../../../tests/qunit/qunit-bootstrap.js"></script>
+	<script type="text/javascript" src="../../../mwEmbedLoader.php"></script>
+	<script type="text/javascript" src="../../../docs/js/doc-bootstrap.js"></script>
+</head>
+<body>
+<h1>ComscoreStreamingTag Plugin - advertising</h1>
```

Comment:
We should provide some details around what is the expected logging behavior. This will help QA and future automated testing efficiently check for regressions. See our DFP ad events API example pages for some detailed expected event flow logging. http://player.kaltura.com/modules/DoubleClick/tests/DoubleClickAdEvents.qunit.html